src/main/java/control/ToolViewController.java

```diff
@@ -0,0 +1,43 @@
+package control;
+
+import gui.ToolButton;
+import javafx.scene.input.MouseEvent;
+
+/**
+ * Controller that manages the tool menu (e.g. shot creation menu).
+ * @author alex
+ */
+public class ToolViewController {
+
+    private ControllerManager controllerManager;
+
+    /**
+     * Constructor.
+     * @param controllerManager Manager of the controllers.
+     */
+    public ToolViewController(ControllerManager controllerManager) {
+        this.controllerManager = controllerManager;
+        initializeTools();
+    }
+
+    /**
+     * Initialize all tools in the toolbox. Adds the corresponding buttons to the views and sets up
+     * the event handlers.
+     */
+    private void initializeTools() {
+        ToolButton blockCreationTool = new ToolButton("Add a block",
+                                                      this.controllerManager
```

Comment:
I'm seeing a lot of lines with few code on it, that makes it less readable in my opinion, but that's a matter of opinion I guess